driver/longhorn

```diff
@@ -230,8 +230,10 @@ formatDevIfNeed() {
             echo -ne "Success"
         fi
     else
-        if ! OUT=$(mkfs -t ${newFsType} ${fullDevName} >/dev/null 2>&1); then
-            echo -ne "Failed to create fs ${newFsType} on device ${fullDevName}"
+        OUT=$(mkfs -t ${newFsType} ${fullDevName} >&1 2>&1)
+        if [ $?!=0 ]; then
```

Comment:
Space between `!=`. And don't use `!=`/`==` on number comparison in bash. Use `-eq` or `-ne`.

Try and see the result of `if [ 0!=0 ]; then echo Zero is not zero; fi`

So, have you tested the positive case yet?